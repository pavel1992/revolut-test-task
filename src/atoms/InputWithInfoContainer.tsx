import styled from 'styled-components';

import { TABLET_BREAKPOINT } from '../styleConstants/mediaConstants';

export const InputWithInfoContainer = styled.div`
  margin-top: 30px;

  @media(max-width: ${TABLET_BREAKPOINT}) {
    margin-top: 0;
  }
`;
