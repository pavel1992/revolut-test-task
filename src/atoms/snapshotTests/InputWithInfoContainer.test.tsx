import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { InputWithInfoContainer } from '../InputWithInfoContainer';


test('renders InputWithInfoContainer correctly', () => {
  const render = renderer
    .create(<InputWithInfoContainer>111</InputWithInfoContainer>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
