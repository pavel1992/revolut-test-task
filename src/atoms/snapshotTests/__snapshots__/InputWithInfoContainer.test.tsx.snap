// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InputWithInfoContainer correctly 1`] = `
.c0 {
  margin-top: 30px;
}

@media (max-width:768px) {
  .c0 {
    margin-top: 0;
  }
}

<div
  className="c0"
>
  111
</div>
`;
